{"uid":"6c9865313f1a211d","name":"test_plugin_objects","fullName":"tests.integration.test_plugins#test_plugin_objects","historyId":"5cbaccc18764e500aba1b97dcf0e3d52","time":{"start":1729042624249,"stop":1729042624249,"duration":0},"description":"Checks if charm gets blocked when trying to disable a plugin in use.","descriptionHtml":"<p>Checks if charm gets blocked when trying to disable a plugin in use.</p>\n","status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 840, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 875, in pytest_runtest_setup\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 160, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1630, in setup\n    self._request._fillfixtures()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 697, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 532, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 617, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1091, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 234, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"tmp_path_factory","time":{"start":1729042496975,"stop":1729042496975,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1729042496975,"stop":1729042497184,"duration":209},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop","time":{"start":1729042496975,"stop":1729042496975,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1729042624250,"stop":1729042624250,"duration":0},"status":"failed","statusMessage":"_pytest.outcomes.XFailed: aborted\n","statusTrace":"  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 234, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"description":"Checks if charm gets blocked when trying to disable a plugin in use.","status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 840, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 875, in pytest_runtest_setup\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 160, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1630, in setup\n    self._request._fillfixtures()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 697, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 532, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 617, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1091, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 234, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","steps":[],"attachments":[{"uid":"64e0030fa1b6db21","name":"log","source":"64e0030fa1b6db21.txt","type":"text/plain","size":2472}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"ops_test::finalizer","time":{"start":1729042624252,"stop":1729042624614,"duration":362},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1729042624614,"stop":1729042624614,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1729042624614,"stop":1729042624614,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1729042624615,"stop":1729042624615,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_plugins"},{"name":"host","value":"runner"},{"name":"thread","value":"17550-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_plugins"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"efca1969f1572bc6","status":"unknown","time":{}}],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":2,"passed":17,"unknown":1,"total":20},"items":[{"uid":"80761d39eeab0738","reportUrl":"../111//#testresult/80761d39eeab0738","status":"passed","time":{"start":1728956603029,"stop":1728956752514,"duration":149485}},{"uid":"2a9b3ddb015d750e","reportUrl":"../110//#testresult/2a9b3ddb015d750e","status":"passed","time":{"start":1728870355686,"stop":1728870498027,"duration":142341}},{"uid":"89bead8e29d73414","reportUrl":"../109//#testresult/89bead8e29d73414","status":"passed","time":{"start":1728783764970,"stop":1728783907362,"duration":142392}},{"uid":"8526919ff2004111","reportUrl":"../108//#testresult/8526919ff2004111","status":"passed","time":{"start":1728697387424,"stop":1728697530927,"duration":143503}},{"uid":"ed494803f3cfc02f","reportUrl":"../107//#testresult/ed494803f3cfc02f","status":"unknown","time":{}},{"uid":"68f46a8160e079f2","reportUrl":"../105//#testresult/68f46a8160e079f2","status":"passed","time":{"start":1728438083349,"stop":1728438217944,"duration":134595}},{"uid":"3797f660ae17605d","reportUrl":"../104//#testresult/3797f660ae17605d","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1728351216098,"stop":1728351216098,"duration":0}},{"uid":"305acf45af11c3ad","reportUrl":"../103//#testresult/305acf45af11c3ad","status":"passed","time":{"start":1728265462754,"stop":1728265597341,"duration":134587}},{"uid":"4f51632b713b2bed","reportUrl":"../102//#testresult/4f51632b713b2bed","status":"passed","time":{"start":1728178954194,"stop":1728179082976,"duration":128782}},{"uid":"7873c820ba556586","reportUrl":"../101//#testresult/7873c820ba556586","status":"passed","time":{"start":1728092490071,"stop":1728092639864,"duration":149793}},{"uid":"c9a6f85a2f1efced","reportUrl":"../100//#testresult/c9a6f85a2f1efced","status":"passed","time":{"start":1728005985496,"stop":1728006132384,"duration":146888}},{"uid":"13b2cfec88b7ec58","reportUrl":"../99//#testresult/13b2cfec88b7ec58","status":"passed","time":{"start":1727919757187,"stop":1727919904251,"duration":147064}},{"uid":"ec9c5d0385edb40","reportUrl":"../98//#testresult/ec9c5d0385edb40","status":"passed","time":{"start":1727833193850,"stop":1727833338232,"duration":144382}},{"uid":"49693d0a52f51ae8","reportUrl":"../97//#testresult/49693d0a52f51ae8","status":"passed","time":{"start":1727747034456,"stop":1727747176769,"duration":142313}},{"uid":"3ebcb3b8289f139c","reportUrl":"../96//#testresult/3ebcb3b8289f139c","status":"passed","time":{"start":1727660663845,"stop":1727660809282,"duration":145437}},{"uid":"968fbd354846d049","reportUrl":"../95//#testresult/968fbd354846d049","status":"passed","time":{"start":1727574189413,"stop":1727574333247,"duration":143834}},{"uid":"5d86bcb777568de3","reportUrl":"../94//#testresult/5d86bcb777568de3","status":"passed","time":{"start":1727487516427,"stop":1727487646803,"duration":130376}},{"uid":"65a17b82eba0f9fe","reportUrl":"../93//#testresult/65a17b82eba0f9fe","status":"passed","time":{"start":1727401442317,"stop":1727401585135,"duration":142818}},{"uid":"d498703c882eb973","reportUrl":"../92//#testresult/d498703c882eb973","status":"passed","time":{"start":1727315117136,"stop":1727315254361,"duration":137225}}]},"tags":["asyncio","group(1)"]},"source":"6c9865313f1a211d.json","parameterValues":[]}